td/tracking/views.py

```diff
@@ -290,13 +478,15 @@ def get(self, request, *args, **kwargs):
             return redirect("tracking:project_list")
 
         allowed_urls = [
-            "http://localhost:8000/tracking/charter/new/",
-            "http://localhost:8000/tracking/event/new/",
-            "http://td.unfoldingword.org/tracking/charter/new/",
-            "http://td.unfoldingword.org/tracking/event/new/",
+            re.compile(r"^/tracking/charter/new/$"),
```

Comment:
Is this also true for regex? I wasn't sure if the slight difference - in the argument part - would break it.